pkgs/development/compilers/sac2c/sac2c-stdlib.nix

```diff
@@ -0,0 +1,82 @@
+{ lib, fetchgit, stdenv, sac2c-unwrapped
+, cmake, git, bison, flex
+, cudaPackages, patchelf
+}:
+
+stdenv.mkDerivation rec {
+  pname = "sac-stdlib";
+  version = "1.3-145-gc52d";
+
+  src = fetchgit {
+    url = "https://github.com/SacBase/Stdlib.git";
+    rev = "c52d4e4f8445cb66bc8c11d1e372770091078a92";
+    sha256 = "sha256-ClJm3anqeX8or//EDaKk1D5KRjPppDrMwUGzdMm1HP4=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
+    bison
+    flex
+    patchelf
+  ];
+
+  buildInputs = [
+    sac2c-unwrapped
+    cudaPackages.cudatoolkit
+  ];
+
+  targets = [ "seq" "mt_pth" "cuda" ];
+
+  prePatch = ''
+    substituteInPlace cmake-common/generate-version-vars.cmake \
+      --replace "\''${GIT_EXECUTABLE} describe --tags --abbrev=4 --dirty" "echo ${version}"
+
+    substituteInPlace cmake-common/misc-macros.cmake \
+      --replace "-DUSER_HOME=\"\''$ENV{HOME}\"" "-DUSER_HOME=$TMPDIR"
+
+    substituteInPlace src/CMakeLists.txt \
+      --replace "MESSAGE (DEBUG" " MESSAGE (STATUS" \
+      --replace \
+        "STRING (REPLACE \"/usr/local/\" \"\" _install_mod_dir \''${INSTALL_MOD_DIR})" \
+        "set(_install_mod_dir \"${placeholder "out"}/lib/modlibs\")" \
+      --replace \
+        "STRING (REPLACE \"/usr/local/\" \"\" _install_tree_dir \''${INSTALL_TREE_DIR})" \
+        "set(_install_tree_dir \"${placeholder "out"}/libexec\")"
+  '';
+
+  preInstall = ''
+    mkdir -p $out/share
+    cp -r $TMPDIR/.sac2crc $out/share/.sac2crc
+  '';
+
+  # For some reason the libs bake in the ephemeral build path instead of the
```

Comment:
Yes, this a design choice to allow for 'local builds'... in particular with all SaC modules. I vaguely remember that @sbscholz, I, and maybe @ashinkarov had discussed moving way from using RPATHs (for this purpose) but it looked like using sac2crc entries might be the solution. At the time though sac2crc was still being progressed for 'package' installations, and not mature yet. Maybe the situations has changed?